docs/guidelines/content-guidelines/04-ui-elements.md

```diff
@@ -0,0 +1,110 @@
+---
+title: UI elements
+---
+
+Contrary to the regular technical documentation, UI elements follow different guidelines and best practices. Due to the limited space you can use to convey a message, your UI text must be as precise and concise as possible. Use the following general guidelines to design and write content for UI elements:
+
+- Use active voice and action-oriented language.
+- Use title case for short one-line texts, such as labels or headings, and sentence case for messages and explanations, unless specified otherwise. (?)
+- Use plural for the object names, as users usually interact with multiple objects within the program, for example:
+
+  ✅ Service Instances <br>
+  ⛔️ Service Instance
+
+- Use plain, natural language. Even though some verbs may be similar in meaning, they may sound more or less natural to the reader. For example, from these pairs of verbs, the first one sounds more natural:
+
+  ✅ Choose <br>
+  ⛔️ Select
+
+  ✅ Edit <br>
+  ⛔️ Modify
+
+- Use the three-letter currency code instead of the currency symbol. This prevents ambiguity when multiple currencies share the same symbol.
+
+  ✅ USD <br>
+  ⛔️ $
+
+
+## Labels
+
+Labels are one-line texts that appear in various types of UI elements, such as buttons, titles and headings, forms, and drop-down menus. When designing labels for the UI elements in the Kyma project, make sure to:
+
+- Use title case for titles and headings.
+- Use sentence case in forms, drop-down menus, and button labels.
```

Comment:
That's in conflict with the [UA Style Guide](https://help.sap.com/docs/SNG_FOR_UA/e33c591ae4494a659a3f5f983c9d1161/79ce282052d14cc38e6a203a1fc277c4.html?state=DRAFT&locale=en-US&q=ui%20elements) and the [Fiori UI Guide](https://experience.sap.com/internal/fiori-design-web/ui-text-guidelines-for-sap-fiori/#formatting) 😐  

> Use title case for labels, titles, and similar short texts on the user interface. Here are some examples: menu items; text on buttons and icons;...

> Unless otherwise specified for individual UI elements, use title case for all SAP Fiori user interface short texts (labels, headings, value help texts, and so on), and sentence case for all messages and explanations.

Reply:
Since Busola wants to conform to Fiori standards, I guess we should follow the Fiori guide.

Reply:
We checked it with @grego952 and in Busola we use sentence case for button labels and drop-down menus. I'll remove "forms" from this bullet point since we use title case for them.

Reply:
But whyyyyy does Busola differ from the Fiori and SAP standards? 😭 What's the benefit, except making it harder to keep consistency?

Reply:
Tbh, I think it just looks better. And I agree with what they say on the UX-related websites that the title case looks more unnatural and not so inviting. Fortunately, we don't have to follow corporate guidelines in OS.